src/main/java/ch/njol/skript/effects/EffActionBar.java

```diff
@@ -54,25 +54,27 @@ public class EffActionBar extends Effect {
 
 	@SuppressWarnings({"unchecked", "null"})
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final SkriptParser.ParseResult parser) {
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parser) {
 		message = (Expression<String>) exprs[0];
 		recipients = (Expression<Player>) exprs[1];
 		return true;
 	}
 
 	@SuppressWarnings("deprecation")
 	@Override
-	protected void execute(final Event e) {
-		String msg = message.getSingle(e);
+	protected void execute(Event event) {
+		String msg = message.getSingle(event);
 		assert msg != null;
```

Comment:
```suggestion
```
bit of a silly assertion

Reply:
Wasn't sure if we wanted to remove it or not